README.md

```diff
@@ -7,6 +7,7 @@ A suite to build, analyze and visualize circuits.
 - [Brayns](#brayns) — Visualizer for large-scale and interactive ray-tracing of neurons.
 - [libsonata](#libsonata) — C++ / Python reader for SONATA circuit files.
 - [BioExplorer](#bioexplorer) - Extract and analyze scientific data for visualization and interactive exploration.
+- [Neurodamus](#neurodamus) - BBP Simulation Control application for NEURON.
```

Comment:
Put it at the top before CoreNeuron as it is a important software

Reply:
Sure, it is done.